### ISSUE TITLE
Dashbord view

### ISSUE THREAD

**Issue by nickknissen** [ThomasBS/domainzter#1]
Something like this ![2_three_columns_view](https://cloud.githubusercontent.com/assets/204970/8179209/76f9421a-1415-11e5-803b-d33624e7dfd0.png)

**Comment by ThomasBS** [comment]
Yeah I was thinking adding the option to change layout (it should also be in the todo-list) - just not figured out how or what. But thanks for suggestion! :v:
